pkg/controller/certificates/certificate_controller.go

```diff
@@ -77,30 +77,31 @@ func NewCertificateController(
 			glog.V(4).Infof("Updating certificate request %s", oldCSR.Name)
 			cc.enqueueCertificateRequest(new)
 		},
-		DeleteFunc: func(obj interface{}) {
```

Comment:
why extract this?

Reply:
I think this moved out from new controller is better from coding view as all other controllers, Which makes code more readable